engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -210,7 +210,7 @@ public static Module synthetic(
   /** Clears any literal source set for this module. */
   public void unsetLiteralSource() {
     this.disposeInteractive();
-    this.sources = ModuleSources.NONE;
+    this.sources = this.sources.reset();
```

Comment:
Please add a unit test (without the protocol machinery) to make sure that `unsetLiteralSource` keeps `file()` reference.